packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1317,6 +1317,9 @@ abstract class State<T extends StatefulWidget> with Diagnosticable {
   /// To artificially cause the entire widget tree to be disposed, consider
   /// calling [runApp] with a widget such as [SizedBox.shrink].
   ///
+  /// To listen for platform shutdown messages (and other lifecycle changes),
+  /// consider the [AppLifecycleListener] API.
```

Comment:
Is it worth explicitly mentioning here that `dispose` may not get called in these cases?

Reply:
That's what the line at the top of this section is trying to say; do you think it needs to call out `dispose` explicitly?

Reply:
Ahh I did not see that in the collapsed diff. I see it now.